packages/components/src/components/modal/_modal.scss

```diff
@@ -106,6 +106,10 @@
     @include carbon--breakpoint(xlg) {
       width: 48%;
     }
+
+    .#{$prefix}--modal-container-body {
+      display: contents;
```

Comment:
Just wanted to double-check, do we need to offer any fallbacks for IE11 support or should this work as expected?

Reply:
@joshblack oh I didn't think about that, are you guys still supporting IE11? the dotcom team has dropped IE from our supported browser list.